backend/src/index.spec.ts

```diff
@@ -1,28 +1,52 @@
 import { resolvers } from './resolvers'
 import { typeDefs } from './typeDefs'
+import { permissions } from './permissions'
 import { PostsDataSource } from './postsDataSource'
 import { ApolloServer } from 'apollo-server'
 import { createTestClient } from 'apollo-server-testing'
+import { applyMiddleware } from 'graphql-middleware'
+import { makeExecutableSchema } from 'graphql-tools'
+const bcrypt = require('bcrypt')
+const jwt = require('jsonwebtoken')
```

Comment:
```suggestion
import bcrypt from 'bcrypt'
import jwt from 'jsonwebtoken'
```
Don't mix ESM with CommonJS import statements.